app/models/conference/importer.rb

```diff
@@ -0,0 +1,73 @@
+require 'csv'
+class Conference::Importer
+  
+  ## This importer is depending on agreed-upon input format.
+  ## Input file:
+  # - Dates should be formatted with dd mm yyyy
+  # - UID should be unique and formatted with 'current season' + id: 2017001
+  # - The headers should not be changed
+  # Output:
+  # Conferences will be updated or created
+  # Conferences deleted in the input file, are not removed from the table
+  # Import errors are logged in a separate 'imports_logger' file
+  
+  class << self
+    def import(file)
+      check_valid(file)
+      info_log(:started, file)
+      process_csv(file)
+      info_log(:finished, file)
+    end
+    
+    private
+    
+    def info_log(arg, file)
+      notices = {
+        started: "Started importing file #{file.original_filename}",
+        finished: "Finished updating/creating #{count_conferences_in(file)} conferences"
+      }
+      imports_logger.info "\n***" + notices[arg] + "***"
+    end
+   
+    def imports_logger
+      Logger.new("#{Rails.root}/log/imports_logger.log")
```

Comment:
Logging to a file isn't really working on Heroku. For one, the log file will get purged due to Heroku's Ephemeral Filesystem logic and secondly, the file will only be available on the dyno the receives the import request – and we can't tell which one that will be. I would just use `Rails.logger` as the logging facility.